csharp/src/Ice/IProtocolPluginFacade.cs

```diff
@@ -10,49 +10,47 @@ public interface IProtocolPluginFacade
         // Get the Communicator instance with which this facade is
         // associated.
         //
-        Ice.Communicator getCommunicator();
+        Ice.Communicator Communicator { get; }
 
         //
         // Register an EndpointFactory.
         //
-        void addEndpointFactory(IEndpointFactory factory);
+        void AddEndpointFactory(IEndpointFactory factory);
 
         //
         // Get an EndpointFactory.
         //
-        IEndpointFactory getEndpointFactory(short type);
+        IEndpointFactory? GetEndpointFactory(short type);
 
         //
         // Obtain the type for a name.
         //
-        System.Type? findType(string name);
+        System.Type? FindType(string name);
     }
 
     public sealed class ProtocolPluginFacade : IProtocolPluginFacade
     {
-        public ProtocolPluginFacade(Ice.Communicator communicator) => _communicator = communicator;
+        public ProtocolPluginFacade(Ice.Communicator communicator) => Communicator = communicator;
```

Comment:
Sounds like IProtolPluginFacade is implemented by a single sealed class, ProtocolPluginFacade ... meaning we should merge them (in a future refactoring)